support-frontend/assets/components/footerCompliant/footerStyles.js

```diff
@@ -2,24 +2,81 @@
 
 import { css } from '@emotion/core';
 import { textSans } from '@guardian/src-foundations/typography';
-import { neutral } from '@guardian/src-foundations/palette';
+import { brand, neutral, brandText, brandAlt } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
 import { space } from '@guardian/src-foundations';
 
 export const componentFooter = css`
-  padding-top: ${space[2]}px;
+  background-color: ${brand[400]};
```

Comment:
Maybe we can use [`brandBackground.primary`](https://www.theguardian.design/2a1e5182b/p/1377a6-tokens/t/63cc10) to be more explicit